src/app/auth/account/account.component.html

```diff
@@ -115,6 +118,7 @@ <h2 class="text-start pt-3 px-3 h6 organization">Organizations</h2>
                         value="Stars"
                         [checked]="activeTab === AccountTabs.STARS"
                         (click)="onSelectStarsTab(user)"
+                        disabled="true"
```

Comment:
I can see you've added these attributes, but the tabs are stil clickable for some reason